addon/templates/components/modal-view.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="modal-dialog {{if centered 'modal-dialog-centered'}} {{size}}" role="document">
+<div class="modal-dialog {{if centered 'modal-dialog-centered'}} {{size}}" role="document" id={{id}}>
```

Comment:
what's this `id` for ? and what's the value of it ?

Reply:
It's to target the modal with the teaser background on the crm repo

Reply:
Ok. can you default it to `null` in the modal-view.js ? just so we know it's a possible attribute to pass